Cargo.toml

```diff
@@ -2,6 +2,15 @@
 name = "hbbft"
 version = "0.1.0"
 authors = ["Vladimir Komendantskiy <komendantsky@gmail.com>"]
```

Comment:
Let's update `authors` too.

Reply:
Should we add all of us individually, or rather a single POA email address?

Reply:
You are right, as the team grows, we might need to follow the example of other projects, e.g.,
```
authors = ["Parity Technologies <admin@parity.io>"]
```

Reply:
@igorbarinov: Is there some email address like that, which we should use as "the author"?

Reply:
I extended the authors list.